src/components/Login/LoginForm.js

```diff
@@ -16,6 +20,20 @@ export default function LoginModal(props) {
   const [waiting, setWaiting] = useState(false);
   const [hoverButton, setHoverButton] = useState(false);
 
+  const handleAnonymousLogin = (e) => {
+    setWaiting(true);
+    setErrorMsg('');
+
+    e.preventDefault();
+    signInAnonymously()
+      .then(() => {})
+      .catch((err) => {
+        console.error(err);
+        setErrorMsg(getCreateUserErrorMessage(err));
```

Comment:
Is this the correct error message?

Reply:
I think so... the link to the error codes for signInAnonymously is broken: https://firebase.google.com/docs/auth/web/anonymous-auth